src/app/app.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { BadgeColour, ContactDetailsDataModel, FilterConfig, TCDocument } from 'projects/exui-common-lib/src';
-import { GoogleTagManagerService } from 'projects/exui-common-lib/src/lib/services/google-tag-manager/google-tag-manager.service';
+import { GoogleTagManagerService} from 'projects/exui-common-lib/src/lib/services/google-tag-manager/google-tag-manager.service';
```

Comment:
I guess the space at the end here was removed by accident?

Reply:
Ahh yes good spot